src/sendMessage.js

```diff
@@ -303,7 +303,7 @@ module.exports = function(defaultFuncs, api, ctx) {
     cb();
   }
 
-  return function sendMessage(msg, threadID, callback) {
+  return function sendMessage(msg, threadID, callback, replyToMessage) {
```

Comment:
The callback is supposed to be optional, but this makes it so that you can't pass a reply without a callback.